test/exercises/2023JanTutorial/kmer_large_input.txt

```diff
@@ -0,0 +1,77 @@
+GAGTTTTATCGCTTCCATGACGCAGAAGTTAACACTTTCGGATATTTCTGATGAGTCGAAAAATTATCTTGATAAAGCAGGAATTACTACTGCTTGTTTACGAATTAAATCGAAGTGGACTGCTGGCGGAAAATGAGAAA
```

Comment:
Is this double-length line intentional / important?

Also:  I thought reading strings broke on whitespace—is it actually processing all the lines in this file or just the first?  Based on a roughtly estimated sum of `nkmerCounts` on my way out the door, it looks like maybe just the first?

Reply:
I fixed the double-length line.  Thanks for noticing this.

Really good catch on the only reading the first line.  Why is that?  On this page (https://chapel-lang.org/docs/modules/standard/IO.html), channel.read does not indicate it breaks at newlines, whereas channel.readline indicates it does.  Is this behavior specific to the string.readThis definition?  How can I read the whole file into a string? (Because then I could strip out the whitespace)

Reply:
I think [readAll](https://chapel-lang.org/docs/latest/modules/standard/IO.html#IO.channel.readAll) will do what you want?

Reply:
readAll did get the whole file.  Then I realized I needed to strip each line, so I read it in line at a time.  Thank you for all the help!

Reply:
> Is this behavior specific to the string.readThis definition?

It is specific to reading strings (which break at any whitespace).  I'm not sure where we document how various types are read by `read` currently...  maybe we don't?